modules/nf-core/seqkit/concat/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "seqkit_concat"
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - "bioconda::seqkit=2.6.1"
```

Comment:
Might you be able to bump this to version 2.7.0?

Reply:
Hi, I've updated it to 2.7.0

However, I'm still struggling to get it passing the tests.

Reply:
Thanks!